content/en/deployment/lambda/index.md

```diff
@@ -0,0 +1,326 @@
+    type: zipFile
+providerOptions:
+  lambda:
+    - name: potatolessFacts
+      target: us-east-1
+      runAsIamRole: arn:aws:iam::111111111111:role/lamdaExecutionRole
+      handler: index.handler
+      runtime: nodejs18.x
+```
+
+## How to trigger a deployment
+
+* [Use the CLI]({{< ref "cli" >}}) in your terminal or with any CI system by installing the CLI natively or running it in Docker.
+* [Use the GitHub Action]({{< ref "integrations/ci-systems/gh-action" >}}) in your GitHub workflow.
+
+
+## Define your deployment
+
+CD-as-a-Service enables you to declare your deployment configuration outcome in a YAML file. 
+
+For an AWS Lambda deployment, you only need to provide the following pieces of information:
+
+* The target you want to deploy to
+* The Lambda function you want to deploy
+* Provider options for that Lambda function
+* The canary strategy you want to use for the deployment
+
+## Elements of a deployment
+
+### Targets
+
+Within a deployment, you define _targets_ that are equivalent to the environments you are deploying to - a unique combination of AWS account (ArmoryRole ARN) and region. 
+
+The following example has multiple region-based targets in a single AWS account:
+
+```yaml
+targets:
+  Prod-East-1:
+    account: armory-prod
+    deployAsIamRole: arn:aws:iam::111111111111:role/ArmoryRole
+    region: us-east-1
+    strategy: allAtOnce
+  Prod-West-2:
+    account: armory-prod
+    deployAsIamRole: arn:aws:iam::111111111111:role/ArmoryRole
+    region: us-west-2
+    strategy: allAtOnce
+```
+
+This example has targets that are separate AWS Accounts:
+
+```yaml
+targets:
+  Lab:
+    account: armory-lab
+    deployAsIamRole: "arn:aws:iam::111111111111:role/ArmoryRole"
+    region: us-west-2
+    strategy: rollingDeployment
+  Staging:
+    account: armory-core
+    deployAsIamRole: "arn:aws:iam::222222222222:role/ArmoryRole"
+    region: us-west-2
+    strategy: rollingDeployment
+  Audit:
+    account: armory-audit
+    deployAsIamRole: "arn:aws:iam::333333333333:role/ArmoryRole"
+    region: us-west-2
+    strategy: rollingDeployment
+  ITSec:
+    account: armory-itsec
+    deployAsIamRole: "arn:aws:iam::444444444444:role/ArmoryRole"
+    region: us-west-2
+    strategy: rollingDeployment
+```
+
+#### Target constraints
+
+You can also configure your deployment targets to use constraints that prevent a deployment from beginning or completing until certain conditions are met. For example, you can configure your deployment to wait for your code to be deployed to your staging environment before promoting that code to production.
+
+You can set `beforeDeployment` or `afterDeployment` constraints depending on your use case. 
+
+CD-as-a-Service offers you multiple constraint options including: 
+
+*  `dependsOn`  
+   Use `dependsOn` to specify a target deployment that must successfully complete prior to starting this target's deployment.
+*  `pause`  
```

Comment:
I find it confusing to talk about 'dependsOn' and 'pause' at the same level. I feel like this should talk about 'beforeDeployment' instead of pause - a beforeDeployment constraint can be used to implement a checklist of things that need to happen before a target starts deploying. For example, it can be used with the pause step to require a manual approval.

I would then add a third bullet for 'afterDeployment' with text like 'An after deployment constraint can be used to prevent downstream deployments from starting until some set of post deployment tasks finishes. For example, this can be used with the runWebhook step to execute a set of end-to-end tests in a staging environment before deploying to production.

Reply:
copied from the existing K8s content, so I'll update that as well